packages/web/src/components/premium-content-purchase-modal/PremiumContentPurchaseModal.module.css

```diff
@@ -15,7 +15,12 @@
   justify-content: center;
 }
 
-.content > div:first-child {
```

Comment:
Curious about this change. The `> div:first-child` selector was working around an issue with `ModalContent` while avoiding affecting some utility divs used by the scrollbar library. So I would be surprised if this didn't introduce some subtle layout issues again.